boards/PDM/Src/Io/Io_Efuses.c

```diff
@@ -0,0 +1,227 @@
+#include "Io_Efuses.h"
```

Comment:
Just looking over this and I feel like it would be better to split Io_Efuses into files containing individual Efuses. Feel like this file might be a bit clunky if we decide to group them all together

Reply:
DONE